src/mscorlib/Common/PinnableBufferCache.cs

```diff
@@ -46,7 +46,7 @@ internal sealed class PinnableBufferCache
 #pragma warning disable 618
         [EnvironmentPermission(SecurityAction.Assert, Unrestricted = true)]
 #pragma warning restore 618
-        [System.Security.SecuritySafeCritical]
```

Comment:
Argh- I see what I did. Copied one of the regexes incorrectly. I'll update and pull the other security attributes as well. I'll put the regex up for everyone to look at.